Add translations to README file

mcruzdev opened the pull request:
This pull request aims to add README.adoc translation to portuguese.

xstefank reviewed (approved):
Thanks @mcruzdev! LGTM

xstefank commented:
@mcruzdev I updated the main README and added individual READMEs to all chapters. Would you be willing to update this PR with the new (final) version please?

mcruzdev commented:
> @mcruzdev I updated the main README and added individual READMEs to all chapters. Would you be willing to update this PR with the new (final) version please?

Of course!

mcruzdev commented:
Working on it (this week)!

gastaldi commented on README.adoc:
```diff
@@ -4,7 +4,7 @@ This repository represents the book code developed in the book link:https://www.
 ---
 
-_Available in_: link:README.adoc[English]
+_Available in_: link:README.adoc[English] | link:README-pt.adoc[Português (Portuguese)]
```
```suggestion
_Available in_: link:README.adoc[English] | link:README-pt.adoc[Português Brasileiro (Brazilian Portuguese)]
```

xstefank reviewed (commented):
Amazing! Thanks @mcruzdev!

And thank you @gastaldi for the reviews!
